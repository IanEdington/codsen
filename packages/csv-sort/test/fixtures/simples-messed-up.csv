Acc Number,Description,Debit Amount,Credit Amount,Balance
123456,Client #1 payment,,1000,1940
123456,Bought table,10,,940
123456,Bought carpet,30,,0
123456,Bought chairs,20,,980
123456,Bought pens,10,,1000