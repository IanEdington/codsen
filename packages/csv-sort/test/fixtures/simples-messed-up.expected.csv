Acc Number,Description,Debit Amount,Credit Amount,Balance
123456,Bought chairs,20,,980
123456,Bought pens,10,,1000